tests/Behat/UserContext/UserOperationsContext.php

```
@@ -204,27 +179,145 @@ public function userWithEmailAndInitialsShouldBeReturned(
+    }
+
+    private function assertContentContains(string $expectedContent): void
+    {
+        $content = $this->getPageContent();
+        $decoded = json_decode($content, true);
+
+        if ($decoded && isset($decoded['detail'])) {
+            Assert::assertStringContainsString(
+                $expectedContent,
+                $decoded['detail']
+            );
```

PHPMD (2.15.0)

Avoid using static access to class '\PHPUnit\Framework\Assert' in method 'assertContentContains'. (Clean Code Rules)

(StaticAccess)